v1/topdown/test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package topdown
+
+import "github.com/open-policy-agent/opa/v1/ast"
+
+const TestCaseOp Op = "TestCase"
+
+func builtinTestCase(bctx BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+	// FIXME: Don't store test-case operands as input doc on event
+	// FIXME: Update Event to include test-case operands in trace logging
```

Comment:
FIXME as in before this feature ships or not terribly important? 🙂

Reply:
Not terribly important, but also just a thing I forgot to touch up. Fixing.